### PR TITLE
[11.0] [ADD] document_page_group

### PR DESCRIPTION
**Pull request by etobella**
This module allows to define groups on a category.

### CODE REVIEW

File: document_page_group/models/document_page.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        relation='document_page_direct_group',
+        column1='document_page_id',
+        column2='group_id',
+        compute='_compute_group_ids'
+    )
+    direct_group_ids = fields.Many2many(
+        'res.groups',
+        relation='document_page_group',
+        column1='document_page_id',
+        column2='group_id',
```
Comment (LoisRForgeFlow) [inline document_page_group/models/document_page.py:25]: I would just a small help for the user, as the names themselves can not be enough

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot and code review, LGTM!

File: document_page_group/models/document_page.py (hunk 1 above)
Comment (etobella) [reply document_page_group/models/document_page.py:25]: Well groups is hidden, but we could define something for direct_group_ids

Comment (LoisRForgeFlow) [reply document_page_group/models/document_page.py:25]: It is not hidden, see in runbot with demo user:
image

Comment (etobella) [reply document_page_group/models/document_page.py:25]: Ups, my mistake then :wink:

Comment (LoisRForgeFlow) [reply document_page_group/models/document_page.py:25]: Now it looks better form the user perspective :+1:

**Comment by etobella** [comment]
@jarroyomorales

**Review by jarroyomorales — APPROVED** [review]
Tested and works as expected! :+1:

**Pull request merged**